.eslintrc.js

```diff
@@ -60,6 +64,7 @@ module.exports = {
         "./src/background/*.ts",
         "./src/electron/*.ts",
         "./tests/**/*.ts",
+        "./build/*",
```

Comment:
あまり問題ない気がするけど、`.txt`とかも含まれているのが意図的なのかどうか判断つかなくなりそうかも。
今はjsしかないけど、こうしとくのどうでしょ。
```suggestion
        "./build/*.js",
        "./build/*.ts",
```

Reply:
確かに。